src/sentry/snuba/discover.py

```diff
@@ -271,6 +271,52 @@ def get_row(row):
     return result
 
 
+# TODO(evanh) This is only here for backwards compatibilty with old queries using these deprecated
+# aliases. Once we migrate the queries these can go away.
+OLD_FUNCTIONS_TO_NEW = {
+    "p75": "percentile(transaction.duration, 0.75)",
+    "p95": "percentile(transaction.duration, 0.95)",
+    "p99": "percentile(transaction.duration, 0.99)",
```

Comment:
I wonder if we shouldn't keep these functions around long term. They are short and to the point, `percentile()` is useful for folks who need more than a simple preset.

Reply:
I think we can come up with a better name at least. `latency_p95` or something just to help convey what these functions operate on.

Reply:
I'll chat with Bowen and nail down what our final list of functions will be for tracing.